app/models/user.rb

```diff
@@ -38,6 +39,7 @@ class User < Common::RedisStore
 
   validates :uuid, presence: true
   validates :email, presence: true
+  validates :loa_current, presence: true
 
   # conditionally validate if user is LOA3
   with_options if: :loa3? do |user|
```

Comment:
it isn't in the UI flow yet but if the data is there we have it being returned...